src/html/scope.rs

```diff
@@ -16,9 +16,9 @@ cfg_if! {
 /// Updates for a `Component` instance. Used by scope sender.
 pub(crate) enum ComponentUpdate<COMP: Component> {
     /// Wraps messages for a component.
-    Message(COMP::Message),
+    Message(Into<COMP::Message>),
     /// Wraps batch of messages for a component.
-    MessageBatch(Vec<COMP::Message>),
+    MessageBatch(Vec<Into<COMP::Message>>),
```

Comment:
@jstarry  I'm not sure about this, since you did not mention changes in `ComponentUpdate` enum